tests/test_auth.py

```diff
@@ -7,17 +8,44 @@
 class TestAuth(AsyncTestCase):
 
     def setUp(self):
-        self.auth_args = ('auth_url', 'username', 'password', 'mock_project_name',
-                          'mock_user_domain_name', 'mock_project_domain_name')
+        self.auth_args = ('http://url', 'm_user', 'm_pass', 'm_project',
+                          'm_user_domain', 'm_project_domain')
         self.auth = AuthPassword(*self.auth_args)
 
     def tearDown(self):
         patch.stopall()
+        for name in list(os.environ.keys()):
```

Comment:
How about context manager for settings envs? 
Maybe something similar to this: https://coderwall.com/p/7h56tq/a-context-manager-for-setting-os-env-in-test-cases

Reply:
undone